src/handlers/meta_transaction_handlers.ts

```diff
@@ -223,8 +228,8 @@ export class MetaTransactionHandlers {
                 price: metaTransactionPrice.price,
                 buyAmount: metaTransactionPrice.buyAmount!,
                 sellAmount: metaTransactionPrice.sellAmount!,
-                sellTokenAddress,
-                buyTokenAddress,
+                sellTokenAddress: isETHSell ? ETH_TOKEN_ADDRESS : sellTokenAddress,
```

Comment:
This can never happen in practice as our meta transactions do not support selling ETH but added it for uniformity between `/swap` and `/meta_transaction`. Don't have any strong opinions here, can remove the change if that is preferred.